Blackfynn/data-portal
# Contact Us Modal

# Description

The purpose of this PR is to add a Contact Us modal to the footer of the page.

## Type of change


# How Has This Been Tested?

1. Scroll down to the footer in the homepage. You should see a `Contact Us` link. Clicking on the link should bring up a modal.
2. Try clicking send before filing out information. Validation should kick off asking you to input data in the required fields.
3. Enter your name, email, and message and click `Send`. The modal should close and the request should have gone through (you can verify this in the "Network" tab).

# Checklist:
